cadctl/cmd/investigate/investigate.go

```diff
@@ -240,72 +240,64 @@ func GetPDClient(webhookPayload []byte) (*pagerduty.SdkClient, error) {
 	return client, nil
 }
 
-// checkCloudProviderSupported takes a list of supported providers and checks if the
-// cluster to investigate's provider is supported
-func checkCloudProviderSupported(cluster *v1.Cluster, supportedProviders []string) (bool, error) {
+func isAWS(cluster *v1.Cluster) (bool, error) {
```

Comment:
Simplified this, I don't think we will need the extensability for now, and it make the code after this more readable.